table-api/src/main/java/io/deephaven/api/updateby/spec/RollingFormulaSpec.java

```diff
@@ -18,9 +17,23 @@
 @BuildableStyle
 public abstract class RollingFormulaSpec extends RollingOpSpec {
 
-    public abstract List<String> formula();
-
-    // Continue single-parameter token support
+    /**
+     * The formula to use to calculate output values. The formula is similar to
+     * {@link io.deephaven.api.TableOperations#update} and {@link io.deephaven.api.TableOperations#updateView} in
+     * specifying the output column name and the expression to compute in terms of the input columns. (e.g.
+     * {@code "outputCol = sum(inputColA + inputColB)"}).
+     * <p>
+     * The alternative (and deprecated) form for {@link #formula()} is active when {@link #paramToken()} is provided. In
+     * this case the formula should only contain the expression in terms of the token. (e.g. {@code sum(x)} where x is
+     * the {@link #paramToken()}). NOTE: This form is deprecated and will be removed in a future release.
+     */
+    public abstract String formula();
+
```

Comment:
Does this change from `List` to a single value require any documentation update? Also, is this a breaking change?

Reply:
Accepting multiple `formula` was never exposed to the user so this change is entirely internal.